packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -301,7 +301,6 @@ Widget _wrapWithBackground({
   );
 
   final bool darkBackground = backgroundColor.computeLuminance() < 0.179;
-  // TODO(jonahwilliams): remove once we have platform themes.
   switch (defaultTargetPlatform) {
     case TargetPlatform.iOS:
       SystemChrome.setSystemUIOverlayStyle(
```

Comment:
shouldn't this be using the new layers?

Reply:
Implemented, but this broke a test and I'm unsure of how to fix